src/locales.py

```diff
@@ -120,6 +120,12 @@ class Locale:
     ChoosePrinterDirectory = "Choose printer name"
     AddNewPrinter = "Add new printer"
     DefaultPrinterWarn = "Be aware that you are using default printer. New data might be removed after update. We recommend to create new printer and calibrate it."
+    CheckUpdates = "Check for updates"
+    ProjectUpdate = "Project update"
+    SettingsUpdate = "The current project settings format is outdated. This may lead to errors in the program. Update format? When updating, the project settings will be reset!"
```

Comment:
They will not be reset if you copy all of them. Just mention that "some new settings will have default values. Please recheck them carefully".